var/spack/repos/builtin/packages/motioncor2/package.py

```diff
@@ -48,3 +48,5 @@ def install(self, spec, prefix):
         mkdirp(prefix.bin)
         for files in glob("MotionCor2_*"):
             install(files, prefix.bin)
+        with working_dir(prefix.bin):
+            symlink('MotionCor2_{}'.format(spec.version), 'MotionCor2')
```

Comment:
You'll need to use `{0}` here as `{}` doesn't work for Python 2.6.

Reply:
Fixed.